qt/python/mantidqt/project/plotsloader.py

```diff
@@ -57,20 +58,32 @@ def make_fig(self, plot_dict, create_plot=True):
         creation_args_copy = copy.deepcopy(creation_args[0])
 
         # Populate workspace names
-        workspace_name = creation_args[0][0].pop('workspaces')
-        workspace = ADS.retrieve(workspace_name)
+        # workspace_name = creation_args[0][0].pop('workspaces')
+        # post_creation_args = creation_args[0][0].pop(MantidAxKwargs.POST_CREATION_ARGS)
+        # workspace = ADS.retrieve(workspace_name)
 
         # Make initial plot
         fig, ax = plt.subplots(subplot_kw={'projection': 'mantid'})
 
+        pcargs = []
+        # If an overplot is necessary plot onto the same figure
+        for cargs in creation_args[0]:
+            workspace_name = cargs.pop('workspaces')
+            pcargs.append(cargs.pop(MantidAxKwargs.POST_CREATION_ARGS, {}))
+            workspace = ADS.retrieve(workspace_name)
+            self.plot_func(workspace, ax, ax.figure, cargs)
+
+        fem = FigureErrorsManager(ax.figure.canvas)
+        for index, pcargs in enumerate(pcargs):
+            if MantidAxKwargs.ERRORS_ADDED in pcargs and pcargs[MantidAxKwargs.ERRORS_ADDED]:
+                if MantidAxKwargs.ERRORS_VISIBLE in pcargs and pcargs[MantidAxKwargs.ERRORS_VISIBLE]:
+                    fem.show_error_bar_for(index)
+                else:
+                    fem.hide_error_bar_for(index)
+
         # Make sure that the axes gets it's creation_args as loading doesn't add them
         ax.creation_args = creation_args_copy
 
-        self.plot_func(workspace, ax, fig, creation_args[0][0])
-
-        # If an overplot is necessary plot onto the same figure
-        self.plot_extra_lines(creation_args=creation_args, ax=ax)
```

Comment:
I don't think I've removed the implementation of this function, it might not be used anymore, and this functionality is merged into the `for cargs in creation_args[0]:` above

Reply:
I've removed plot_extra_lines and the other comments